### DISCUSSION TITLE
[all] Transcendence architecture

### DISCUSSION THREAD

**Discussion by JuneParkCode** [3DPong/transcendence#15] [Show and tell]
---- FIX
![image](https://user-images.githubusercontent.com/81505228/229086467-87bb41d6-7901-48b7-868b-9b7b9191c9ef.png)

---- 이전 버전
![image](https://user-images.githubusercontent.com/81505228/224909735-59149f6c-f75d-45fd-b4fb-68e12714799a.png)

local 환경에서 개략적으로 어떻게 상호작용 중인지 보여주는 도식입니다.
참고용으로 활용하세요~

**Comment by JuneParkCode** [comment]
https://www.figma.com/file/0M9H27xk2oEsdjklmu4fw4/pong-structure?node-id=0%3A1

**Comment by kyu-baek** [comment]
감사합니다~

**Comment by JuneParkCode** [comment]
23.03.31
변경된 아키텍쳐 반영 및 #80  을 통한 확정
